src/conditionals/programs_with_decision_points/hard/q14/checkWord.java

```diff
@@ -0,0 +1,58 @@
+package conditionals.programs_with_decision_points.hard.q14;
+/*
+Write a program that asks the user for two words, word1 and word2.
+Then, check if the combined length of both words:
+Less than 10: Print "Short phrase."
+Between 10 and 15 (inclusive): Print "Medium phrase."
+Greater than 15: If word1 starts with lowercase "a", print "a-start long phrase."
+Otherwise, print "Long phrase."
```

Comment:
```suggestion
Write a program that asks the user for two words, word1 and word2. Then check if the combined length of both words is less than 10, if so print "Short phrase." If it's between 10 and 15 (inclusive) print "Medium phrase." If it is greater than 15 and word1 starts with lowercase "a", print "a-start long phrase." Otherwise, print "Long phrase."
```

Reply:
This question doesn't feel set up in a way to prompt students to use a switch statement, the fact that even the solution initially uses an if statement tells me this needs to get reworked a bit.

Reply:
That's fair, I forgot to write it but it's a Parsons which I thought would help. Plus giving the chronological directions of first finding the length I thought I would help. Also this is a good way to use if statements and switch statements together which I thought might be nice since it's the hardest switch category.